src/app/link/link.go

```diff
@@ -87,7 +87,8 @@ func Link(cCtx *cli.Context) error {
 		mappers = append(mappers, dic)
 	}
 
-	mappers = append(mappers, mapper.Github{})
+	githubMapper := mapper.WithLeadingVCheck(mapper.Github{})
+	mappers = append(mappers, githubMapper)
```

Comment:
Given that this feature requires network access, I think we should at least make it opt-out so it can run on restricted environments.

Reply:
New option included here: [8161372](https://github.com/newrelic/release-toolkit/pull/134/commits/81613724263fb3cc885d3ad6e1594b11a47c6cee)